scripts/install-template.mjs

```diff
@@ -70,3 +86,16 @@ console.log('==> Assert application version');
 assert.strictEqual(applicationPkgJson.version, '1.0.0');
 console.log('==> Assert dependency versions to not use "workspace:" protocol');
 assert.doesNotMatch(applicationPkgJsonRaw, /workspace:/);
+
+console.log('==> Assert application config file exists');
+if (applicationType === 'custom-application') {
+  assert.strictEqual(
+    doesFileExist(path.join(applicationPath, 'custom-application-config.mjs')),
+    true
+  );
+} else if (applicationType === 'custom-view') {
+  assert.strictEqual(
+    doesFileExist(path.join(applicationPath, 'custom-view-config.mjs')),
```

Comment:
Nit: we can simply use `existsSync` (we use it many places in the repo), so there is no need to have a try/catch

```suggestion
    fs.existsSync(path.join(applicationPath, 'custom-application-config.mjs')),
    true
  );
} else if (applicationType === 'custom-view') {
  assert.strictEqual(
    fs.existsSync(path.join(applicationPath, 'custom-view-config.mjs')),
```

Reply:
I used this implementation as it's the one recommended in Nodejs docs since `fs.exists` is a [deprecated function](https://nodejs.org/docs/latest-v18.x/api/fs.html#fsexistspath-callback).

![image](https://github.com/commercetools/merchant-center-application-kit/assets/97907068/dfd38599-5805-410b-ae61-6ec3101fe7d7)

Reply:
🤦‍♂️ My bad, sorry.

I guess we would need to refactor the other places then to stop using `fs.existsSync`.
I'll create a task.